Workbooks/Resource Insights/AzureDataExplorerCluster/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Azure Data Explorer Cluster Resource Overview",
+    "author": "Microsoft",
+    "isPreview": true,
+    "galleries": [{ "type": "workbook", "resourceType": "microsoft.kusto/clusters"}]
```

Comment:
We might need to add the insights type also `{ "type": "adxcluster-insights", "resourceType": "microsoft.kusto/clusters", "order": 100 }`